src/main/java/technology/tabula/CommandLineApp.java

```diff
@@ -118,11 +248,46 @@ public boolean accept(File dir, String name) {
 
         for (File pdfFile : pdfs) {
             File outputFile = new File(getOutputFilename(pdfFile));
-            extractFileInto(pdfFile, outputFile);
+            try{
+                extractFileInto(pdfFile, outputFile);
+            }
+            catch(ParseException pe){
+                System.out.println(pe.getMessage());
```

Comment:
Do not write to stdout. If you need to report progress, or any other message, use loggers.